spec/queries/find_exact_query_spec.rb

```diff
@@ -290,8 +314,21 @@
     expect(query.exact_match).to be_nil
   end
 
+  it "matched on content_id only" do
+    query = build_query(content_id: "11a51f5e-3f17-4516-87c7-02afba87ef40")
+    subscriber_list = create_subscriber_list(content_id: "11a51f5e-3f17-4516-87c7-02afba87ef40")
+    expect(query.exact_match).to eq(subscriber_list)
+  end
+
+  it "not matched on different content_id" do
+    query = build_query(tags: { policies: { any: %w[beer] } }, content_id: "11a51f5e-3f17-4516-87c7-02afba87ef40")
```

Comment:
Remove the `tags`, as currently I think this is testing the same thing as the tags + content_id matching above - but really it should just be testing the content_id.